x-pack/plugins/security_solution/server/plugin.ts

```diff
@@ -625,6 +628,10 @@ export class Plugin implements ISecuritySolutionPlugin {
           );
 
           await turnOffAgentPolicyFeatures(fleetServices, productFeaturesService, logger);
+
+          // Ensure policies have backing DOT indices (We don't need to `await` this.
+          // It can run in the background)
+          ensureIndicesExistsForPolicies(this.endpointAppContextService).catch(() => {});
```

Comment:
Is there a specific reason for leaving this catch block empty? Shouldn’t we at least log the error?

Reply:
Its to silence an ESLint rule that requires all promises to be handled.